src/stm.py

```diff
@@ -136,13 +123,26 @@ def _cbf_welch(volume, fps, max_freq = 20):
     N = nextpow2(volume.shape[0])
     f, Pxx = signal.welch(volume, fs = fps, window = "hann", nfft = N,
         return_onesided = True, axis = 0)
-    max_freq_indices = Pxx.argmax(axis = 0)
-    heatmap = f[max_freq_indices]
-    heatmap[heatmap > max_freq] = max_freq
-    return heatmap
+    return get_best_volumes(Pxx,f,max_freq,sequence)
 
 # Utilities
 
+def get_best_volumes(pxx, f, max_freq, top = 5):
+    max_freq_indices = np.array(- pxx).argsort(axis = 0)[:top]
+    heatmap_list = list();
+    for i in range(0, len(max_freq_indices)):
+        heatmap = f[max_freq_indices[i]]
+        heatmap[heatmap > max_freq] = max_freq
+        heatmap_list.append(heatmap)
+    return np.array(heatmap_list)
+
```

Comment:
Spurious newlines between end of this function and header of next function. Again, per PEP guidelines, should only be 1 blank line between function definitions.